examples/apps/presence-tracker/src/app.ts

```diff
@@ -74,17 +76,19 @@ async function start() {
 
 	initializeReactions(presence, mouseTracker);
 
-	const focusDiv = document.getElementById("focus-content") as HTMLDivElement;
+	const focusDiv = document.querySelector("#focus-content") as HTMLDivElement;
 	renderFocusPresence(focusTracker, focusDiv);
 
-	const mouseContentDiv = document.getElementById("mouse-position") as HTMLDivElement;
+	const mouseContentDiv = document.querySelector("#mouse-position") as HTMLDivElement;
 	renderMousePresence(mouseTracker, focusTracker, mouseContentDiv);
 
-	const controlPanelDiv = document.getElementById("control-panel") as HTMLDivElement;
+	const controlPanelDiv = document.querySelector("#control-panel") as HTMLDivElement;
 	renderControlPanel(mouseTracker, controlPanelDiv);
 
 	// Setting "fluid*" and these helpers are just for our test automation
-	const buildAttendeeMap = () => {
+	// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+	const buildAttendeeMap = (): {} => {
```

Comment:
Could this be a `Record<something, something>`?

Reply:
Yeah, good call. Updated.